### PR TITLE
Address default values for response content types 

### PR DESCRIPTION
**Pull request by sebastiankb**
This issue addresses #611
- it remove ExpectedResponse from the [default table ](https://w3c.github.io/wot-thing-description/#sec-default-values)
- it change the assignment from `with default` to `mandatory` in the ExpectedResponse [table](https://w3c.github.io/wot-thing-description/#expectedresponse)
- setup a new assertion to define the assumption that response content type == form.contentType when input and output content type are equal
- make examples

### CODE REVIEW

File: validation/td-validation.ttl [hunk 1]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages.
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: can be used to provide metadata for the expected response message.
With the core Vocabulary, it only includes content type information,
but TD Context Extensions could be applied.

File: validation/td-validation.ttl [hunk 2]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages. 
+<span class="rfc2119-assertion" id="td-expectedResponse-default-contentType">
+If no <code>response</code> name-value pair is provided it MUST be assumed that the content type of the
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: ... provided, (comma)

File: validation/td-validation.ttl [hunk 3]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
 operations types SHOULD NOT be arbitrarily set by servients
 </span>.
 </p>
+
+<p>
+The optional <code>response</code> name-value pair can be used to provide metadata, such as content type information, that is only itended for the reponse messages. 
+<span class="rfc2119-assertion" id="td-expectedResponse-default-contentType">
+If no <code>response</code> name-value pair is provided it MUST be assumed that the content type of the 
+response is equal to the assigned content type of the form.</span>
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: ... to the content type assigned to the Form instance.

File: validation/td-validation.ttl [hunk 4]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
...
+E.g., if the value of the content type of the form is <code>application/xml</code> the assumed value of the content type of the response will be also <code>application/xml</code>.
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: "E.g.," should never be used in the beginning of a sentence.
Use "For instance," instead.

File: validation/td-validation.ttl [hunk 5]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
...
+</p>
+
+
 <p>
-In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type for the <code>input</code> 
-and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
+In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type 
+for the <code>input</code> and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: ... and output data.

File: validation/td-validation.ttl (hunk 4 above)
Comment (mkovatsc) [inline validation/td-validation.ttl]: Before the "For instance", best add the hint:
Note that contentType within an ExpectedResponse Class does not have a Default Value.

File: validation/td-validation.ttl [hunk 6]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
...
 <span class="rfc2119-assertion" id="td-expectedResponse-contentType">
-If the content type of the expected response does not equal its <a>Default Value</a> and differs from the content type of the form,
-the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>.
+If the content type of the expected response differs from the content type of the form,
+the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>. 
+E.g., the <code>input</code> of an <code>ActionAffordance</code> accept <code>application/json</code> as content type and its
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: This should be outside of the rfc2119 span.

For instance, ...

Comment (mkovatsc) [reply validation/td-validation.ttl]: ... An `ActionAffordance` could only accept `application/json` for its input data, while it
will respond with an `image/jpeg` content type for its output data.

File: validation/td-validation.ttl [hunk 7]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
...
+In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type 
+for the <code>input</code> and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
 <span class="rfc2119-assertion" id="td-expectedResponse-contentType">
-If the content type of the expected response does not equal its <a>Default Value</a> and differs from the content type of the form,
-the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>.
+If the content type of the expected response differs from the content type of the form,
+the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>. 
+E.g., the <code>input</code> of an <code>ActionAffordance</code> accept <code>application/json</code> as content type and its 
+corresponding <code>output</code> will response <code>image/jpeg</code> as content type. In that case the content types differe and the
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: differ (no e at the end)

File: validation/td-validation.ttl [hunk 8]
```diff
@@ -367,15 +367,31 @@ well-known types necessary to implement the WoT interaction model described in
...
+In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type 
+for the <code>input</code> and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
 <span class="rfc2119-assertion" id="td-expectedResponse-contentType">
-If the content type of the expected response does not equal its <a>Default Value</a> and differs from the content type of the form,
-the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>.
+If the content type of the expected response differs from the content type of the form,
+the <code>Form</code> instance MUST include a name-value pair with the name <code>response</code>. 
+E.g., the <code>input</code> of an <code>ActionAffordance</code> accept <code>application/json</code> as content type and its 
+corresponding <code>output</code> will response <code>image/jpeg</code> as content type. In that case the content types differe and the 
+<code>response</code> name-value pair has to be used to provide the response content type (<code>image/jpeg</code>) information
```
Comment (mkovatsc) [inline validation/td-validation.ttl]: .. provide response content type information (image/jpeg)

(no "the", example value at the end)

File: index.html [hunk 9]
```diff
@@ -1405,23 +1405,43 @@ <h2>Hypermedia Controls Vocabulary Definitions</h2>
...
+For instance, if the value of the content type of the form is <code>application/xml</code> the assumed value of the content type of 
+the response will be also <code>application/xml</code>.
+</p>
+
+
 <p>
-In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type for the <code>input</code> 
-and <code>output</code>. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
+In some use cases such as for an <code>ActionAffordance</code> there is a different expectation about the content type 
+for the <code>input</code> and <code>output</code> data. In such a case, the optional <code>response</code> name-value pair can describe the content type of the expected response.
```
Comment (mkovatsc) [inline index.html]: Sorry, kein code env, da sonst das DataSchema gemeint ist, nicht die Daten

**Comment by egekorkan** [comment]
The latest commit brought the content of PR #658 , thank you!

**Pull request merged**